### ISSUE TITLE
Update from homebridge?

### ISSUE THREAD

**Issue by nerdherdinc** [estruyf/homebridge-presence-switch-msgraph#19]
Is it possible to use this to set presence status, rather than just reading it? I'd like to be able to set my status via homekit or via a StreamDeck, or when it changes in Slack, have it change on Teams, etc... basically integrating multiple presence systems.

**Comment by estruyf** [comment]
Is it possible? Not at the moment, this would be a great enhancement. 

About the multiple presence systems. Not sure how many are using this, but could also be a great enhancement.

**Comment by BobboSA** [comment]
Getting Admin consent for a .write permission on even a small tenant would be a challenge, until they allow user consent in the API I suspect this may be a blocker for most.

**Comment by estruyf** [comment]
Fair point, two app registrations would be a solution for this. One that supports read/write, the other only read. It is up to the user and company policies to choose.
